rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -105,22 +100,22 @@ function LandingPageGraph() {
         setShowMVO(showDisplay);
     };
 
-    function toggleBundleAdj_PointCloud(showDisplay) {
-        /*Toggles the display of the Bundle Adjustment Point Cloud.
+    function toggleDA_PointCloud(showDisplay) {
+        /*Toggles the display of the Data Association Point Cloud.
 
         Args:
             showDisplay (boolean): Sets the display to be shown or not.
         */
         setShowDA_PC(showDisplay);
     };
 
-    function toggleRotSummaryDisplay(showDisplay) {
-        /*Toggles the display of the rotation averaging metrics.
+    function toggleBA_PointCloud(showDisplay) {
+        /*Toggles the display of the Bundle Adjustment Point Cloud.
 
         Args:
             showDisplay (boolean): Sets the display to be shown or not.
         */
-        setShowRSS(showDisplay);
```

Comment:
what does RSS stand for? and did we remove the `toggleRotSummaryDisplay` function?

Reply:
I ended up removing it because it was an old variable that wasn't needed anymore. RSS just stood for "rotation success summary", but now that's being shown when you click on the plate.